packages/components-angular/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts

```diff
@@ -142,8 +142,6 @@ export class SwissPostIntranetHeaderComponent implements OnInit, OnChanges, Afte
       this.moreElement = this.dom.nativeElement.querySelector('#more');
       const toggleElement = this.moreElement.getElementsByTagName('span')[0];
       toggleElement.addEventListener('click', () => this.toggleMenuOverflow());
-      // TODO: Discuss this behavior, it should be consistent throughout all dropdowns (nav/more/language)
-      // moreElement.addEventListener('blur', () => this.hideMenuOverflow());
```

Comment:
You know better than me if this is in a ticket or already handled? Or perhaps I should just ignore it in Sonar?